packages/fluentui/accessibility/src/behaviors/MenuButton/menuButtonBehavior.ts

```diff
@@ -15,7 +15,7 @@ export const menuButtonBehavior: Accessibility<MenuButtonBehaviorProps> = props
     attributes: {
       trigger: {
         'aria-controls': props.open ? props.menuId : undefined,
-        'aria-expanded': props.open || undefined,
+        'aria-expanded': (props.open && !props.contextMenu) || undefined,
```

Comment:
I think it's nit but wouldn't be less DOM polluting if we do add anything?

```suggestion
        ...(!props.contextMenu && { 'aria-expanded': props.open && }),
```

Reply:
We don't want to have `aria-expanded="false"`, if `props.open` is `false`, so we want to keep `undefined` part :)

Reply:
So in the suggestions we would be not having `aria-exapended` at all if `contextMenu` is true it wouldn't be `aria-expanded="false"` it would be nothing.

Reply:
Does `aria-expanded={undefined}` actually render on the DOM, AFAIK it shouldn't

Reply:
@assuncaocharles  your suggestion makes sense to me:

no context menu -> always undefined

Reply:
@ling1726 you are right, `aria-expanded={undefined}` will not render on the DOM. so doesn't matter, resolving this comment